docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+# Updating the capability level of a channel
+
+*Audience: network administrators, node administrators*
+
+If you're not familiar with capabilities, check out [Capabilities](./capabilities_concept.html)
+before proceeding, paying particular attention to the fact that **node versions
+must be upgrading to the required level for all affected nodes before enabling
+capabilities**.
+
```

Comment:
**peers and orderers that belong to the channel must be upgraded before enabling capabilities**